libs/MeshKernel/include/MeshKernel/BilinearInterpolationOnGriddedSamples.hpp

```diff
@@ -30,10 +30,19 @@
 #include <MeshKernel/Mesh2D.hpp>
 #include <MeshKernel/MeshInterpolation.hpp>
 
+#include <concepts>
+#include <span>
+
 namespace meshkernel
 {
+
+    template <typename T>
+    concept InterpolatableType =
+        std::same_as<T, short> ||
+        std::same_as<T, float>;
+
     /// @brief A class for performing bilinear interpolation on gridded samples
-    template <typename ValueType>
+    template <typename InterpolatableType>
     class BilinearInterpolationOnGriddedSamples : public MeshInterpolation
```

Comment:
I do not think that Concepts work like this.

The concept definition is correct, but its use in the template is incorrect. Change to (The Scalar name can be changed to whatever you want)

template<InterpolatableType Scalar>

Reply:
Was too quick to push. Fixed.